Atomic/rpmwriter.py

```diff
@@ -0,0 +1,369 @@
+import os
```

Comment:
Do you need to have a shebang here declaring the version of python you're supporting?  Perhaps:

#!/usr/bin/env python2

Given your python3 issue noted below?